### DISCUSSION TITLE
Use helm to build Milvus under the k8s cluster. After installing the milvus cluster, the pod node is unhealthy

### DISCUSSION THREAD

**Discussion by itwangxiaobai** [milvus-io/milvus#26613] [Q&A and General discussion]
the log output is:
Warning  Unhealthy  7m54s (x14 over 10m)    kubelet            Readiness probe failed: HTTP probe failed with statuscode: 500
 Warning  Unhealthy  2m50s (x11 over 9m50s)  kubelet            Liveness probe failed: HTTP probe failed with statuscode: 500

![image](https://github.com/milvus-io/milvus/assets/50311975/0b5813d8-7da0-467a-b7a3-4b37a9c7a112)

**Comment by yhmo** [comment]
@LoveEachDay might can help

**Comment by LoveEachDay** [comment]
@itwangxiaobai We use `:9091/healthz` as health probe. `datacoord` relies on `rootcoord`, could you provide all the pods log, using this [script](https://github.com/milvus-io/milvus/blob/master/deployments/export-log/export-milvus-log.sh)?
